aiosdnotify/__init__.py

```diff
@@ -36,29 +46,31 @@ def __init__(self, debug=False, warn_limit=3, watchdog=True):
         self._warnings = 0
         self._transport: asyncio.DatagramTransport | None = None
         self._protocol: asyncio.DatagramProtocol | None = None
-        self._watchdog_task: asyncio.Task | None = None
+        self._watchdog_task: asyncio.Task[Any] | None = None
```

Comment:
I could not figure out what the proper type parameter for the Task generic is here.

Reply:
I think it's just the return type of the function in the task. In this case I think it's just `None`?

Reply:
That would make sense